iotile_cloud/utils/mock_cloud.py

```diff
@@ -301,6 +306,14 @@ def _get_streamer_ack(self, device_id, index):
 
         return 0
 
+    def device_extra(self, request, obj_id):
+        """Handle /device/<slug>/extra/ API that returns stream counts."""
+
+        base = self.one_object('devices', request, obj_id).copy()
```

Comment:
This is not technically correct, as the `extra` does not currently return everything the base returns, but it is probably not worth writing the extra code to filter out what is not included

Reply:
Got it.  I didn't realize that.  I'll update to only keep the right keys.